app/models/ability.rb

```diff
@@ -401,9 +401,9 @@ def stockit_abilities
 
   def user_abilities
     can [:current_user_profile], User
-    can [:show, :update, :orders_count], User, id: @user_id
-    can [:index, :show, :update, :recent_users, :create], User if can_read_or_modify_user?
-    can [:create, :show], User if can_create_donor?
+    can %i[show update orders_count mentionable_users], User, id: @user_id
```

Comment:
We should restrict the `mentionable_users` action to only users that are allowed to use it.

```
can %i[mentionable_users], User if can_mention_users?
```

Give the `can_mention_users` permission to the roles for Sys Admin, Reviewer, Supervisor, Order Admin..., Order Fulfillment